app/main/views/services.py

```diff
@@ -506,6 +507,76 @@ def update_section_submission(framework_slug, lot_slug, service_id, section_id,
                                 service_id=service_id))
 
 
+@main.route('/frameworks/<framework_slug>/submissions/<lot_slug>/<service_id>/remove/<section_id>/<question_slug>',
+            methods=['GET', 'POST'])
+@login_required
+def remove_subsection(framework_slug, lot_slug, service_id, section_id, question_slug):
+    try:
+        draft = data_api_client.get_draft_service(service_id)['services']
+    except HTTPError as e:
+        abort(e.status_code)
+
+    if not is_service_associated_with_supplier(draft):
+        abort(404)
+
+    content = content_loader.get_manifest(framework_slug, 'edit_submission').filter(draft)
+    question_to_remove = content.get_question_by_slug(question_slug)
+    containing_section = get_containing_section(content, question_to_remove)
+    fields_to_remove = question_to_remove.form_fields
+
+    if request.args.get("confirm") and request.method == "POST":
+        # Remove the section
+        for field in fields_to_remove:
+            draft[field] = None
```

Comment:
I don't think we should be submitting the full draft as an update – we should be sending the question fields with `None` values only, since it's the only thing we're changing. The API should be able to figure out the rest. The only thing we could do here is check if none of the fields are in the draft and then skip sending the request, but since we hide the delete button and there's no harm in sending the request we can safely skip that.